bundles/org.openhab.binding.yioremote/src/main/java/org/openhab/binding/yioremote/internal/utils/Websocket.java

```diff
@@ -61,7 +63,27 @@ public void onConnect(Session session) {
     public void onError(Throwable cause) {
         logger.warn("WebSocketError {}", cause.getMessage());
         if (websocketHandler != null) {
-            websocketHandler.onError();
+            websocketHandler.onError(cause);
+        }
+    }
+
+    @OnWebSocketClose
+    public void onClose(int statusCode, String reason) {
+        if (statusCode != StatusCode.NORMAL) {
+            logger.debug("WebSocket Connection closed: {} - {}", statusCode, reason);
+        }
+        if (session != null) {
+            if (!session.isOpen()) {
+                if (session != null) {
+                    session.close();
+                }
+            }
+            session = null;
+        }
+
+        if (websocketHandler != null) {
+            websocketHandler.onClose();
+            logger.warn("Debug2");
```

Comment:
Remove

Reply:
Why should I remove this?

Reply:
Because it's a completely useless debug message at a very high log level.

Reply:
Correct haven't seen the debug message :(

Reply:
solved